ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -164,6 +165,19 @@ export const isBypassed = (
 ): boolean =>
   !!getAccessModuleStatusByName(profile, moduleName)?.bypassEpochMillis;
 
+export const displayModuleStatus = (
+  profile: Profile,
+  moduleName: AccessModule
+): string => {
+  const moduleStatus = computeRenewalDisplayDates(
+    getAccessModuleStatusByName(profile, moduleName)
+  ).moduleStatus;
+  const canModuleExpire = getAccessModuleConfig(moduleName).canExpire;
+  return !canModuleExpire && moduleStatus === AccessModulesStatus.EXPIRED
+    ? AccessModulesStatus.COMPLETE
```

Comment:
I'd prefer a "cannot expire" or something like that instead

Reply:
so the reason i did not do that was because that would be redundant since the expiration date also has 'NEVER EXPIRE'

Reply:
as we discussed offline, a "Cannot Expire" status would also hide whether the module is complete or not, which we don't want